pkg/sqlqueries/SQL/system_intake/get_by_user.sql

```diff
@@ -1,6 +1,8 @@
-SELECT *
-FROM system_intakes
+SELECT DISTINCT si.*
```

Comment:
`DISTINCT` is not really needed here, contacts are constrained to be added to an intake only once.